target/linux/ath79/dts/qca9563_rosinson_wr818.dts

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	model = "ROSINSON WR818";
+	compatible = "rosinson,wr818";
+
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &system;
+		led-failsafe = &system;
+		led-running = &system;
+		led-upgrade = &system;
+	};
+
+	gpio_leds: leds {
+		compatible = "gpio-leds";
+
+		system: system {
+			label = "rosinson:red:system";
```

Comment:
LED name scheme should be {board}:{color}:{name} . The only exception in ath79 is TP-LINK because there are tons of TP-LINK boards sharing similar LED configurations.